code/modules/religion/rites.dm

```diff
@@ -747,3 +747,68 @@
 	..()
 	to_chat(user, span_nicegreen("You feel [GLOB.deity]'s tenacity pouring into you!"))
 	user.AddElement(/datum/element/tenacious)
+
+
+///exorcism rite
+
+/datum/religion_rites/exorcism //In the future I would love to turn exorcism (and also posession) into a component or something more versatile, but for now this works.
+	name = "Exorcism"
+	desc = "The first thing they teach you in Chaplain College. Expels hostile spirits, demonic influence, and any other junk that may be floating around in someone's soul."
```

Comment:
this is really misleading. anyone's first guess is gonna be this curing cultists